kcidb/monitor/output.py

```diff
@@ -136,9 +138,26 @@ def render(self):
             ready to send, but missing the From header.
         """
         email = EmailMessage()
-        email["Subject"] = self.message.summary + self.obj.summarize()
+        subject = self.message.summary + self.obj.summarize()
+        email["Subject"] = subject
         email["To"] = ", ".join(self.message.recipients)
         email["X-KCIDB-Notification-ID"] = self.id
         email["X-KCIDB-Notification-Message-ID"] = self.message.id
-        email.set_content(self.message.description + self.obj.describe())
+        message = self.message.description + self.obj.describe()
+        email.set_content(message)
+        escaped_subject = html.escape(subject)
+        escaped_message = html.escape(message)
+        urls = re.compile(r'((http|https|git|ftp)?://[^|\s]+)')
+        escaped_message = urls.sub('<a href= "\\1">\\1</a>', escaped_message)
+        html_msg = textwrap.dedent("""\
+                        <html>
+                            <head>
+                                <title>{}</title>
+                            </head>
+                            <body>
+                                <pre>{}</pre>
+                            </body>
+                        </html>
+                    """).format(escaped_subject, escaped_message)
+        email.add_alternative(html_msg, subtype='html')
```

Comment:
While you still haven't submitted an amended version, could you please reduce the indent of the string so the closing quote is on the same column as the start of the statement? E.g.:
```suggestion
        html_msg = textwrap.dedent("""\
            <html>
                <head>
                    <title>{}</title>
                </head>
                <body>
                    <pre>{}</pre>
                </body>
            </html>
        """).format(escaped_subject, escaped_message)
        email.add_alternative(html_msg, subtype='html')
```

Reply:
@spbnick, I have made the changes you requested.